packages/server/src/aes-encryptor.tsx

```
@@ -0,0 +1,93 @@
+    const envKeyPath = path.resolve(process.cwd(), 'storage', 'environment-variables', appName, 'aes_key.dat');
+    const envKeyExists = await fs.exists(envKeyPath);
+    if (envKeyExists) {
+      return envKeyPath;
+    }
+    return appKeyPath;
+  }
+
+  static async create(app: Application) {
+    let key: any = process.env.APP_AES_SECRET_KEY;
+    if (!key) {
+      const keyPath = await this.getKeyPath(app.name);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)